js-miniapp-sdk/src/miniapp.ts

```diff
@@ -118,36 +120,28 @@ export interface UserInfoProvider {
 
 /** @internal */
 class UserInfo implements UserInfoProvider {
-  private bridge: MiniAppBridge;
-
-  constructor(miniAppBridge: MiniAppBridge) {
-    this.bridge = miniAppBridge;
-  }
-
   getUserName(): Promise<string> {
-    return this.bridge.getUserName();
+    return (window as any).MiniAppBridge.getUserName();
   }
 
   getProfilePhoto(): Promise<string> {
-    return this.bridge.getProfilePhoto();
+    return (window as any).MiniAppBridge.getProfilePhoto();
```

Comment:
Using `any` will make us lose the typing benefits of TypeScript, so we don't know whether we're accidentally trying to use a function on `MiniAppBridge` which doesn't exist. 

Could you instead create a function which dynamically returns the typed `MiniAppBridge`? For example:

```
function getBridge() {
  return (window as any).MiniAppBridge as MiniAppBridge;
}
```

Reply:
Yes, I have updated.